api/tests/routes/native/v1/offerers_test.py

```diff
@@ -3,12 +3,17 @@
 from pcapi.connectors.acceslibre import ExpectedFieldsEnum as acceslibre_enum
 import pcapi.core.offerers.factories as offerer_factories
 from pcapi.core.offerers.models import VenueTypeCode
+from pcapi.core.testing import assert_num_queries
 
 
 pytestmark = pytest.mark.usefixtures("db_session")
 
 
 class VenuesTest:
+    expected_num_queries = (
+        5  # venue + venue + google_places_info + venue_contact + accessibility_provider + opening_hours
```

Comment:
Le détail fait 6, est-ce une seule fois `venue` ?
```suggestion
        5  # venue + google_places_info + venue_contact + accessibility_provider + opening_hours
```